src/AltinnCore/Common/Services/Implementation/WorkflowSI.cs

```diff
@@ -97,17 +100,67 @@ public ServiceState MoveServiceForwardInWorkflow(Guid instanceId, string owner,
         }
 
         /// <inheritdoc/>
-        public string GetUrlForCurrentState(Guid instanceId, string owner, string service, WorkflowStep currentState)
+        public ServiceState MoveServiceForwardInWorkflow(Guid instanceId, string applicationOwnerId, string applicationId, int instanceOwnerId)
         {
-            switch (currentState)
+            //string developer = AuthenticationHelper.GetDeveloperUserName(_httpContextAccessor.HttpContext);
```

Comment:
Is commented code really necessary? Should it be removed?

Reply:
Removed